### PR TITLE
Remove rickroll

### PR DESCRIPTION
**Pull request by hello-smile6**
Sorry, I'll make a pull request to fix it on every fork. This PR also updates dependencies in `package-lock.json` (no breaking changes).

### CODE REVIEW

**Comment by hello-smile6** [comment]
@JaydenDev Could you merge this?

**Comment by webdev03** [comment]
I think this is a sign that we should always review what code goes into our projects - you know, what if someone put a keylogger instead?

**Comment by hello-smile6** [comment]
> I think this is a sign that we should always review what code goes into our projects - you know, what if someone put a keylogger instead?

I agree. I was mainly hoping it wouldn't get merged. I noticed that my previous PRs were merged without much (or any, in fact) review.

**Comment by webdev03** [comment]
@JaydenDev please merge to avoid rick astley

**Comment by hello-smile6** [comment]
> @JaydenDev please merge to avoid rick astley

I've already made PRs to fix this for @Polygon , just make PRs for everyone else.

**Comment by JaydenDev** [comment]
don't be silly with PRs.

**Comment by hello-smile6** [comment]
> don't be silly with PRs.

I really did add a rickroll! This removes it.

**Comment by webdev03** [comment]
No @JaydenDev We are being serious, PLEASE REVIEW THE PR, see there is actually a rickroll that overrides native keybinds.

File: app/main/main.js [hunk 1]
```diff
@@ -21,7 +22,8 @@ function createWindow () {
 //      click: () => {  }
 //    }]
 //  }));
-  app.whenReady().then(() => {
```
Comment (hello-smile6) [inline app/main/main.js:24]: @JaydenDev See?

**Review by hello-smile6 — COMMENTED** [review]
Please merge. We are serious.

**Comment by webdev03** [comment]
No need to merge now, I did in my PR I think

**Pull request closed without merging**